src/util.ts

```diff
@@ -59,3 +70,39 @@ export function sumBy<T>(arr: T[], fn: (i: T) => number): number {
 export function capitalize(s: string): string {
   return s ? s.charAt(0).toUpperCase() + s.slice(1).toLowerCase() : ''
 }
+
+export const dirExists = async (input: string): Promise<string> => {
+  if (!fs.existsSync(input)) {
+    throw new Error(`No directory found at ${input}`)
+  }
+
+  if (!(await fs.promises.stat(input)).isDirectory()) {
+    throw new Error(`${input} exists but is not a directory`)
+  }
+
+  return input
+}
+
+export const fileExists = async (input: string): Promise<string> => {
+  if (!fs.existsSync(input)) {
+    throw new Error(`No file found at ${input}`)
+  }
+
+  if (!(await fs.promises.stat(input)).isFile()) {
+    throw new Error(`${input} exists but is not a file`)
+  }
+
+  return input
+}
+
+export function isTruthy(input: string): boolean {
+  return ['true', 'TRUE', '1', 'yes', 'YES', 'y', 'Y'].includes(input)
```

Comment:
This cannot handle `True` | `Yes`. I would suggest testing the lower case of input